src/SSHLibrary/pythonclient.py

```diff
@@ -160,11 +164,10 @@ def _get_file(self, remote_path, local_path):
         self._client.get(remote_path, local_path)
 
     def _absolute_path(self, path):
-        path = path.encode(self._encoding)
         if not self._is_windows_path(path):
             path = self._client.normalize(path)
-        if not isinstance(path, unicode):
-            path = unicode(path, self._encoding)
+        if not is_string(path):
+            path = unic(path)
```

Comment:
Should use `is_unicode`, not `is_string` here as well. Also, should use `path.decode(self._encoding)` instead of blindly using `unic`.